file/file.go

```diff
@@ -67,6 +70,11 @@ type MigrationFiles []MigrationFile
 func (f *File) ReadContent() error {
 	if len(f.Content) == 0 {
 		content, err := ioutil.ReadFile(path.Join(f.Path, f.FileName))
+
+		r := bytes.NewBuffer(content)
+ 		line, err := r.ReadString('\n')
```

Comment:
Honestly we could just do a https://golang.org/pkg/bytes/#Contains

so

```
if bytes.Contains(content, []byte("-- @Notransactions")){
}
```
of HasPrefix if we want to anchor it to the start of the file.